lib/flutter/flutter_daemon.dart

```diff
@@ -465,3 +467,30 @@ class LogMessage {
 
   String toString() => '[$level] $message';
 }
+
+typedef Future PerformRequest();
+
+/// A [Job] implementation to wrap calls to the daemon server.
+class DaemonRequestJob extends Job {
+  final PerformRequest _fn;
+
+  DaemonRequestJob(String name, this._fn) : super(toTitleCase(name));
+
+  bool get quiet => true;
+
+  Future run() {
+    return _fn().catchError((e) {
+      if (!analysisServer.isActive) return null;
```

Comment:
Sorry for my lack of knowledge here, but...
How is analysisServer.isActive related to a daemon request job?
Why does it suppress the notification?